src/libraries/System.Memory/tests/MemoryMarshal/CreateReadOnlySpanFromNullTerminated.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using System.Runtime.InteropServices;
+
+namespace System.SpanTests
+{
+    public static partial class MemoryMarshalTests
+    {
+        [Theory]
+        [InlineData(0)]
+        [InlineData(1)]
+        [InlineData(256)]
+        public static unsafe void CreateReadOnlySpanFromNullTerminated_Char(int length)
```

Comment:
Consider changing these unit tests to use `BoundedMemory`. Since the implementation deals with unmanaged memory, this type helps detect read buffer overruns in the implementation. It has previously caught regressions. See [the string ctor tests](https://github.com/dotnet/runtime/blob/1bfdab43807670a7fba468d09b9afd1a72d1f8e3/src/libraries/System.Runtime/tests/System/StringTests.cs#L48-L92) for an example of how we use it there.

Reply:
Done